root/src/style/components/example.scss

```diff
@@ -1 +1,57 @@
- 
+$solidBlueBorder: 3px solid #27d2e7;
+$backgroundColor:#fffdf8;
+
+
+@mixin circle($size){
+  border-left: $solidBlueBorder;
+  //TODO:
+  // longer spinning tail if needed
+  //EX:
+  // border-bottom: $solidBlueBorder;
+  //
+  background: $backgroundColor;
+  padding: $size + px $size + px;
+  width: 0;
+  border-radius: $size + px;
+
+  -webkit-animation: loading 1s infinite linear;
+  -moz-animation: loading 1s infinite linear;
+  -o-animation: loading 1s infinite linear;
+  animation: loading 1s infinite linear;
+}
+
+@-webkit-keyframes loading {
+  from {
+    -webkit-transform: rotate(0deg);
+  }
+  to {
+    -webkit-transform: rotate(360deg);
+  }
+}
+@-moz-keyframes loading {
+  from {
+    -moz-transform: rotate(0deg);
+  }
+  to {
+    -moz-transform: rotate(360deg);
+  }
+}
+@-o-keyframes loading {
+  from {
+    -o-transform: rotate(0deg);
+  }
+  to {
+    -o-transform: rotate(360deg);
+  }
+}
+@keyframes loading {
+  from {
+    transform: rotate(0deg);
+  }
+  to {
+    transform: rotate(360deg);
+  }
+}
+.circularProgress__overlay{
```

Comment:
Let's stick to a naming convention for class names. So this should be `circular-progress-overlay` instead of `circularProgress__overlay`. Find a better and more suggestive name for the class name